2023-round-1/devotuoma/nx_tutorial_script.py

```diff
@@ -0,0 +1,57 @@
+# importing networkx library
+import networkx as nx
+# printing the version once more
+print(nx.__version__)
+G = nx.DiGraph() # calling on the diagraph function using G as the global variable
+# Graph is always a collection of nodes along with identified pairs of nodes i.e. edges, links, e.t.c
+NetworkX, nodes can be any hashable object such as text string, an image, an XML object, another Graph, a customized node object, etc.
+#incorporating nodes with attributes group and color based on types
+G.add_node(1)
+G.add_node(1, group='int',color='red')
```

Comment:
Here you add node 1 twice. The resulting graph will not have 2 nodes labeled "1" but with different attributes.